charts/grafana/templates/_pod.tpl

```diff
@@ -561,6 +561,9 @@ containers:
 {{ toYaml .Values.livenessProbe | indent 6 }}
     readinessProbe:
 {{ toYaml .Values.readinessProbe | indent 6 }}
+{{- if .Values.grafana.lifecycleHooks }}
```

Comment:
Should be .Values.lifecycleHooks.

==> Linting charts/grafana
Error:  templates/: template: grafana/templates/deployment.yaml:49:10: executing "grafana/templates/deployment.yaml" at <include "grafana.pod" .>: error calling include: template: grafana/templates/_pod.tpl:564:14: executing "grafana.pod" at <.Values.grafana.lifecycleHooks>: nil pointer evaluating interface {}.lifecycleHooks

Reply:
Whoops my bad :)